ROADMAP.md

```diff
@@ -56,7 +56,7 @@ Planned\upcoming cmdlets.
 | Get-PowerBIGatewayForDataset | Discover gateways |
 | New-PowerBIDatasetEmbedToken | Generate an embed token |
 | Get-PowerBIDatasetRefreshHistory | Get refresh history |
-| Get-PowerBITablesInDatasource | Get tables in data source |
+| Get-PowerBITable | Get tables in dataset |
```

Comment:
You can probably remove this from the roadmap since you've finished the cmdlet.

Reply:
Got it